packages/agoric-cli/test/agops-perf-smoketest.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
```

Comment:
Why not `#!/usr/bin/env zx`?  That would save many imports. I suppose you'd prefer those imports for being explicit about ambient authority. What is the risk that you're trying to mitigate by that?

Reply:
OCap discipline isn't just about mitigating risk. It's about testability, auditability, modularity, reusability. In sum: good engineering.

Reply:
Engineering is all about trade-offs. One not in your list is velocity.

Reply:
I find it's good for velocity too, especially if done always from the beginning so you don't have to retrofit later.